core/config/docs/core.toml

```
@@ -651,3 +651,19 @@ TransmitQueueMaxSize = 10_000 # Default
 # when sending a message to the mercury server, before aborting and considering
 # the transmission to be failed.
 TransmitTimeout = "5s" # Default
+
+[Beholder]
```

@pkcll @patrickhuie19 Does it actually make sense to call this `Beholder`? Or is that an implementation detail? Or a subsection, only if necessary? What if we unified things here under plain `Telemetry`:
```suggestion
[Telemetry]
```

---

Agreed that to external NOPs `Beholder` has no meaning. I like `Telemetry`, but we also have `TelemetryIngress` today

---

Yep, agree +1 for "[Telemetry]"

---

> Agreed that to external NOPs `Beholder` has no meaning. I like `Telemetry`, but we also have `TelemetryIngress` today

True, but it will eventually be replaced by this, right? Or is that not straightforward? I would think we could document it well enough, but maybe that is too confusing.

---

Yea, good point, it should be replaced by this. +1 for `Telemetry` from me

---

And corresponding configs for Tracer,Meter,Logger,Emitter will be 
[Telemetry.Tracer]
or [Telemetry.Tracing]
...
[Telemetry.Meter] 
... 
?

---

Updated to `[Telemetry]` but I didn't add any subsections yet. There is only one field that would fit at the moment: `TraceSampleRatio`. I think I am ok with prefixes instead of sub-sections, but I suppose that too many of them would be messy. I'm not sure how to predict how this will grow :thinking: